Auth check for valid token #234

rock16 opened the pull request:
Endpoints that require authentication should not allow a user to access them without a valid token that was provided by the login endpoint. the middleware checks the token against the secret key and signing algorithm. issue #234 `/v1/welcome` is protected so you need an authorization token to access it.

samsoft00 commented:
@rock16 why so many changes in your PR? even on someone else code?
Can you make it separate PR please

@Aminujb please note.

rock16 commented:
Initially during the merge I accepted both incoming and mine. I deleted mine so the incoming moved up. That's why it appears like I changed the other. Looking at the file you will see there is no difference between what appears to be my code vs the deleted code
